AssetManagementSystem/appsettings.json

```diff
@@ -7,6 +7,10 @@
   },
   "AllowedHosts": "*",
   "ConnectionStrings": {
-    "dbstring": "Server=;Database=AssetManegementOfficial;Trusted_Connection=True;TrustserverCertificate=True"
+
+    "dbstring": "Server=;Database=AssetManegementOfficial4;Trusted_Connection=True;TrustserverCertificate=True"
```

Comment:
lets keep this as a empty string 

`dbstring: ""`

Reply:
done